fastlane/lib/fastlane/documentation/actions_list.rb

```diff
@@ -73,7 +73,17 @@ def self.show_details(filter: nil)
       else
         puts "Couldn't find action for the given filter.".red
         puts "==========================================\n".red
+
         print_all # show all available actions instead
+
+        if !filter.nil? && filter.length > 1
+          candidates = []
+          all_actions(nil) do |a, action_name|
+            candidates << [action_name] if action_name.include? filter
```

Comment:
ah, that was a leftover from previous code, fixed it as it was no longer necessary.